stream_alert_cli/terraform/athena.py

```diff
@@ -29,23 +29,36 @@ def generate_athena(config):
     athena_dict = infinitedict()
     athena_config = config['lambda']['athena_partition_refresh_config']
 
-    data_buckets = set()
-    for refresh_type in athena_config['refresh_type']:
-        data_buckets.update(set(athena_config['refresh_type'][refresh_type]))
+    data_buckets = list(set(athena_config['buckets']))
+
+    prefix = config['global']['account']['prefix']
+    database = athena_config.get('database_name', '{}_streamalert'.format(prefix))
+
+    results_bucket_name = athena_config.get('results_bucket', '').strip()
+    if results_bucket_name == '':
```

Comment:
You may just use `if not results_bucket_name:`.  The empty string is treated as false anyway.

Reply:
Oh good catch on this - I missed updating this one :)